src/main/java/io/confluent/connect/hdfs/HdfsSinkConnectorConfig.java

```diff
@@ -190,6 +190,12 @@
   public static final String PARTITION_FIELD_NAME_DEFAULT = "";
   public static final String PARTITION_FIELD_NAME_DISPLAY = "Partition Field Name";
 
+  public static final String PARTITION_TIME_FIELD_NAME_CONFIG = "partition.timefield.name";
```

Comment:
Normally we'd separate each word in the config name with a `.` (as you separate each in the variable name with a `_`), so `"partition.time.field.name"` would probably be better here.